matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -1,15 +1,19 @@
-use std::pin::Pin;
-
 use futures::{future::Fuse, Future, FutureExt, StreamExt};
 use futures_channel::mpsc::{UnboundedReceiver, UnboundedSender};
 use futures_util::select;
 use log::debug;
-
+use messages::*;
+use std::pin::Pin;
+use uuid::Uuid;
 mod messages;
 mod signal_peer;
 
+/// The duration, in milliseconds, to send "Keep Alive" packets
```

Comment:
```suggestion
/// The duration, in milliseconds, to send "Keep Alive" requests
```

Matches terminology used in the rest of the project.